plugins/modules/ec2_key.py

```diff
@@ -47,8 +47,18 @@
       - rsa
       - ed25519
     version_added: 3.1.0
+  file_name:
+    description:
+      - Name of the file where the generated private key will be saved.
+      - When provided, the I(key.private_key) attribute will be removed from the return value.
+      - The file is written out on the 'host' side rather than the 'controller' side.
+      - Ignored when I(state=absent) or I(key_material) is provided.
+    type: path
+    version_added: 7.0.0
 notes:
   - Support for I(tags) and I(purge_tags) was added in release 2.1.0.
+  - For security reasons, this module should be used with B(no_log=true) and (register) functionalities
+    when creating new key pair without providing key_material.
```

Comment:
I guess it is worth to move this to a separate PR, so that it can be backported to some more stable branches. (My assumption is that new features don't go to all still maintained stable branches, if that's wrong please someone correct me.)